app/repositories/queue_repository.rb

```
@@ -156,6 +156,7 @@ def update_decass_record(decass_record, decass_attrs)
 
     def update_location_to_attorney(vacols_id, attorney)
       vacols_case = VACOLS::Case.find(vacols_id)
+      fail VACOLS::Case::InvalidLocationError, "Invalid location \"#{attorney.vacols_uniq_id}\"" unless attorney.vacols_uniq_id
```

hm this will only fail when attorney.vacols_uniq_id is nil or false. is that what you want?

---

Yep! `update_vacols_location!` used to do nothing in this case, so this is the case in which I want to throw an error. Not sure whether blank locations are valid, so I don't want to ban those